src/components/app/details/cdDetails/CDDetails.tsx

```diff
@@ -99,6 +99,12 @@ export default function CDDetails() {
                     ?.deploymentAppType,
             )
         }
+        if(result && result[0]){
+            let _selectedEnvironment=  (result[0]['value']?.result || []).find((envData) => {
+                return +envId===envData.environmentId
```

Comment:
Please format the selection and combine the result checks for 96 & 102 together.